docs/sources/reference/commandline/cli.md

```diff
@@ -1656,8 +1666,10 @@ log entry. To ensure that the timestamps for are aligned the
 nano-second part of the timestamp will be padded with zero when necessary.
 
 The `--since` option shows logs of a container generated only after
-the given date, specified as RFC 3339 or UNIX timestamp. The `--since` option
-can be combined with the `--follow` and `--tail` options.
+the given date, specified as RFC 3339 date, UNIX timestamp or a Go duration
```

Comment:
The `--since` option shows only the container logs generated after a given date. You can specify the date as an RFC 3339 date, a UNIX timestamp, or a Go duration string (e.g. `1m30s`, `3h`).  Docker computes the date relative to the client machine's time settings.  You can combine the `--since` option with either or both of the  `--follow` or `--tail` options.